scripts/system/create/entityList/html/js/entityList.js

```diff
@@ -1660,14 +1679,63 @@ function loaded() {
                     } else {
                         document.getElementById("hmdmultiselect").style.display = "none";                      
                     }
+                } else if (data.type === "loadedConfigSetting") {
+                    if (typeof(data.defaultRadius) === "number") {
+                        elFilterRadius.value = data.defaultRadius;
+                        onRadiusChange();
+                    }
+                    if (data.columnsData !== "NO_DATA" && typeof(data.columnsData) === "object") {
+                        var isValid = true;
+                        var originalColumnIDs = [];
+                        for (let originalColumnID in COLUMNS) {
+                            originalColumnIDs.push(originalColumnID);
+                        }                        
+                        for (let columnSetupIndex in data.columnsData) {
+                            var checkPresence = originalColumnIDs.indexOf(data.columnsData[columnSetupIndex].columnID);
+                            if (checkPresence === -1) {
+                                isValid = false;
+                                break;
+                            }
+                        }
+                        if (isValid) {
+                            for (var columnIndex = 0; columnIndex < data.columnsData.length; columnIndex++) {
+                                if (data.columnsData[columnIndex].data.alwaysShown !== true) {
+                                    var columnDropdownID = "entity-table-column-" + data.columnsData[columnIndex].columnID;
+                                    if (data.columnsData[columnIndex].width !== 0) {
+                                        document.getElementById(columnDropdownID).checked = false;
+                                        document.getElementById(columnDropdownID).click();
+                                    } else {
+                                        document.getElementById(columnDropdownID).checked = true;
+                                        document.getElementById(columnDropdownID).click();
+                                    }
+                                }
+                            }
+                            for (columnIndex = 0; columnIndex < data.columnsData.length; columnIndex++) {
+                                let currentColumnIndex = originalColumnIDs.indexOf(data.columnsData[columnIndex].columnID);
+                                if (currentColumnIndex !== -1 && columnIndex !== currentColumnIndex) {
+                                    for (var i = currentColumnIndex; i > columnIndex; i--) {
+                                        swapColumns(i-1, i);
```

Comment:
```suggestion
                                        swapColumns(i - 1, i);
```
And similarly in the next two lines.